build/dsc/templates/dsc_type_spec.rb.erb

```diff
@@ -162,6 +162,12 @@ describe Puppet::Type.type(:dsc_<%= resource.friendlyname.downcase %>) do
   end
 <%      end #  @spec_test_values['munged_ints']... -%>
 
+<%    elsif property.int_array? || property.int? || property.uint? || property.uint_array?  # if property.int? -%>
```

Comment:
This is the only way I could short circuit the existing int/uint logic. Int and uint arrays should accept single values. but can't accept a string. Is there a better way?